### ISSUE TITLE
Custom loss gradient calculation

### ISSUE THREAD

**Issue by Arminius4** [manifoldai/mf-eng-public#1]
Hey,
In [the custom loss function notebook for lightgbm](https://github.com/manifoldai/mf-eng-public/blob/master/notebooks/custom_loss_lightgbm.ipynb) ln[16], there is the line `grad = np.where(residual<0, -2*10.0*residual, -2*residual)`. 
Why is the gradient negative here? The corresponding question on the medium article has not been answered yet and I stumbled upon the same issue. 
Could you please explain why it doesn't work if the "mathematically correct" gradient is returned?

Best regards,
Lukas
